compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h

```diff
@@ -237,18 +355,18 @@ inline void AddElementwise(int size, const BinaryArithmeticOpParam &params,
     auto a21 = vld1q_f32(input2_data + i + 4);
     auto a22 = vld1q_f32(input2_data + i + 8);
     auto a23 = vld1q_f32(input2_data + i + 12);
-    auto x0 = vaddq_f32(a10, a20);
-    auto x1 = vaddq_f32(a11, a21);
-    auto x2 = vaddq_f32(a12, a22);
-    auto x3 = vaddq_f32(a13, a23);
-    x0 = vmaxq_f32(activation_min, x0);
-    x1 = vmaxq_f32(activation_min, x1);
-    x2 = vmaxq_f32(activation_min, x2);
-    x3 = vmaxq_f32(activation_min, x3);
-    x0 = vminq_f32(activation_max, x0);
-    x1 = vminq_f32(activation_max, x1);
-    x2 = vminq_f32(activation_max, x2);
-    x3 = vminq_f32(activation_max, x3);
+    auto x0 = FUNC::Op4(a10, a20);
+    auto x1 = FUNC::Op4(a11, a21);
+    auto x2 = FUNC::Op4(a12, a22);
+    auto x3 = FUNC::Op4(a13, a23);
```

Comment:
@krayzemli Could you please use more descriptive name instead of `FUNC` and `Op4`?